package.json

```diff
@@ -4,8 +4,8 @@
     "postinstall": "node script/get-quasar.js",
     "clean": "rimraf output && rimraf .pulp-cache && rimraf test/tmp",
     "build": "pulp build -- --censor-lib --strict",
-    "property-tests": "pulp test --test-path test/src --main Test.Property.Main",
-    "test": "pulp test --test-path test/src --main Test.Property.Main && pulp test --test-path test/src"
+    "test-fast": "pulp test --test-path test/src --main Test.Fast",
```

Comment:
I guess I just don't think the distinction is an important one - the tests are the tests, and they all should pass. When you want to run them repeatedly during development you use the ones that finish quickly, when you're done with whatever the thing is you run the full tests. I'm not sure making the task name more descriptive has any real advantage. I'd be happy to rename `Test.Fast` though, as that is a weird name for sure. :smile: